spec/features/user/dashboard_spec.rb

```diff
@@ -10,18 +10,73 @@
     expect(page).to have_no_content("GitHub")
   end
 
-  it 'can visit dashboard and see GitHub repos' do
+  it 'can visit dashboard and see GitHub repos', :js do
     user = User.create({email: "fake@example.com",
                         first_name: "David",
                         last_name: "Tran",
                         password: "password",
                         role: "default",
-                        git_hub_token: "297f3266de9167cd907402888af4721c431bb1dc"})
+                        git_hub_token: ENV['GH_TEST_KEY_1']})
 
     allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
 
     visit dashboard_path
 
-    expect(page).to have_content("GitHub")
+    within('#repos') do
+      expect(page).to have_content("GitHub")
+      expect(page).to have_link("brownfield-of-dreams")
+      expect(page).to have_link("neos")
+      click_link("brownfield-of-dreams")
+    end
+
+    expect(current_url).to eq("https://github.com/jrsewell400/brownfield-of-dreams")
+  end
+
+  it "users only see repos associated with their unique token" do
+    user1 = User.create({email: "fake@example.com",
+                        first_name: "David",
+                        last_name: "Tran",
+                        password: "password",
+                        role: "default",
+                        git_hub_token: ENV['GH_TEST_KEY_1']})
+
+    user2 = User.create({email: "example@example.com",
+                        first_name: "Jordan",
+                        last_name: "Sewell",
+                        password: "password",
+                        role: "default",
+                        git_hub_token: ENV['GH_TEST_KEY_2']})
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user1)
+
+    visit dashboard_path
+
+    expect(page).to have_content("test-repo")
```

Comment:
I created a `test-repo` for this test.. Since our program sorts it by the top 5 most updated repos.. My GH key, displays it whereas Jordan's does not